environs/config/config.go

```diff
@@ -432,22 +432,22 @@ var defaultConfigValues = map[string]interface{}{
 	NetBondReconfigureDelayKey: 17,
 	ContainerNetworkingMethod:  "",
 
-	"default-series":             jujuversion.SupportedLTS(),
-	ProvisionerHarvestModeKey:    HarvestDestroyed.String(),
-	ResourceTagsKey:              "",
-	"logging-config":             "",
-	AutomaticallyRetryHooks:      true,
-	"enable-os-refresh-update":   true,
-	"enable-os-upgrade":          true,
-	"development":                false,
-	"test-mode":                  false,
-	TransmitVendorMetricsKey:     true,
-	UpdateStatusHookInterval:     DefaultUpdateStatusHookInterval,
-	EgressSubnets:                "",
-	FanConfig:                    "",
-	CloudInitUserDataKey:         "",
-	ContainerInheritProperiesKey: "",
-	BackupDirKey:                 "",
+	"default-series":              jujuversion.SupportedLTS(),
+	ProvisionerHarvestModeKey:     HarvestDestroyed.String(),
+	ResourceTagsKey:               "",
+	"logging-config":              "",
+	AutomaticallyRetryHooks:       true,
+	"enable-os-refresh-update":    true,
+	"enable-os-upgrade":           true,
+	"development":                 false,
+	"test-mode":                   false,
+	TransmitVendorMetricsKey:      true,
+	UpdateStatusHookInterval:      DefaultUpdateStatusHookInterval,
+	EgressSubnets:                 "",
+	FanConfig:                     "",
+	CloudInitUserDataKey:          "",
+	ContainerInheritPropertiesKey: "",
```

Comment:
What's the chances anyone scripts against this?

Reply:
I think it should be OK; this is the variable name after all. The value is unchanged.